miden-crypto/src/merkle/smt/full/mod.rs

```diff
@@ -254,6 +254,11 @@ impl Smt {
         })
     }
 
+    /// Returns an iterator over the inner nodes of this [Smt].
+    pub fn inner_node_indices(&self) -> impl Iterator<Item = (NodeIndex, InnerNode)> + '_ {
```

Comment:
I would update the doc comment here because otherwise it looks identical to the doc comment of `inner_nodes()`.